packages/market-service/src/exchange-rates-host/exchange-rates-host.ts

```diff
@@ -1,16 +1,44 @@
 import { HistoryData, HistoryTimeframe, MarketData } from '@shapeshiftoss/types'
-import dayjs from 'dayjs'
+import dayjs, { Dayjs } from 'dayjs'
 
 import { FiatMarketService } from '../api'
 import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
-import { FiatMarketDataArgs, FiatPriceHistoryArgs } from '../fiat-market-service-types'
+import {
+  FiatMarketDataArgs,
+  FiatPriceHistoryArgs,
+  SupportedFiatCurrencies
+} from '../fiat-market-service-types'
 import { bnOrZero } from '../utils/bignumber'
 import { rateLimitedAxios } from '../utils/rateLimiters'
 import { ExchangeRateHostHistoryData, ExchangeRateHostRate } from './exchange-rates-host-types'
 
 const axios = rateLimitedAxios(RATE_LIMIT_THRESHOLDS_PER_MINUTE.DEFAULT)
 const baseCurrency = 'USD'
 
+export const makeUrls = (
```

Comment:
I'd prefer to see this called something more specific like `makeExchangeRateRequestUrls` or something.